### ISSUE TITLE
Fails to work

### ISSUE THREAD

**Issue by Mayurifag** [excid3/superleggera#1]
I have an application with webpacker, vue and all the stuff. Also I have buefy framework for css and wanted to integrate purgecss. There is a slim package for views, so I have `/app/views/layouts/application.html.slim`, if that matters. 

It's kinda hard to whitelist everything manually for purgecss, so I found this gem. But it seems not to work for me.

I have a `JSON::ParserError 751: unexpected token at '{["test"]` issue.

Stacktrace says that it's parsing whitelist path but I didn't create whitelist.json anywhere. I found, that it scans gem path and found that whitelist.json has content like `{['test]`, without closing brace, so it's not correct json.

Then I deleted this file and now I got undefined method `scan' for #<ActionDispatch::Response::RackBody:0x00007f4bc0073cb8> error,

Its on method **parse_classes** at `lib/superleggera.rb` ; line 28.

My development.js is copy-paste from documentation, I just changed .erb to .slim.

What can I do to make it work? If the gem is not production-ready, I can be a beta tester, if its needed.

**Comment by excid3** [comment]
Yeah the gem is just barely started. It currently watches your Rails requests in development and starts building up a whitelist of classes used in your Rails app. That way it can try to catch the dynamic classes that are hard to parse out of your helpers, etc.

It still needs support to analyze Javascript, but it's a start at least.

I haven't tested any of this with slim, and don't really plan to since I don't use it.

The whitelist file is generated at `config/whitelist.json`. Your whitelist file should just be a JSON array, not an object. Yours appears to be an object, that also isn't closed. I'd just simply delete that file and let the gem recreate it.

There's about a million other issues and features this needs to work well, the main features being the ones in the TODO section in the readme.
